library/alloc/src/vec/mod.rs

```diff
@@ -2141,7 +2141,9 @@ impl<T, A: Allocator> Vec<T, A> {
     unsafe fn split_at_spare_mut_with_len(
         &mut self,
     ) -> (&mut [T], &mut [MaybeUninit<T>], &mut usize) {
-        let Range { start: ptr, end: spare_ptr } = self.as_mut_ptr_range();
```

Comment:
Yeah this is like a few other things around `Vec` that I fixed, which all boiled down to implicitly creating a slice and then using that to do something in the uninit part.